### PR TITLE
Modified ckcp/run.sh to use kustomize instead of git clone and applying patch files for pipelines

### PR DESCRIPTION
**Pull request by bnallapeta**
Added a variable to refer to kcp kubeconfig - **KUBECONFIG_KCP**

Local Run:

```
./run.sh pipelines
./test.sh pipelines
```

```
[bnr@bnr ckcp]$ k -n ckcp get pods
NAME                    READY   STATUS    RESTARTS   AGE
ckcp-76ff95585f-chh79   1/1     Running   0          3m14s
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ k -n pipelines get pods
NAME                         READY   STATUS    RESTARTS   AGE
pipelines-57785c787b-t775g   1/1     Running   0          100s
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ k get ns | grep -i kcp
ckcp                                                          Active   3m36s
kcpe2cca7df639571aaea31e2a733771938dc381f7762ff7a077100ffad   Active   55s
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ k -n kcpe2cca7df639571aaea31e2a733771938dc381f7762ff7a077100ffad get pods
NAME                               READY   STATUS      RESTARTS   AGE
custom-env-px8m4-pod               0/1     Completed   0          61s
test-pipelinerun-bsfl5-task1-pod   0/3     Completed   0          59s
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ 
[bnr@bnr ckcp]$ KUBECONFIG=work/kubeconfig/admin.kubeconfig k get ns
NAME               STATUS   AGE
default            Active   79s
kube-public        Active   3m41s
kube-system        Active   3m42s
tekton-pipelines   Active   2m25s
[bnr@bnr ckcp]$ KUBECONFIG=work/kubeconfig/admin.kubeconfig k -n tekton-pipelines get cm
NAME                     DATA   AGE
config-artifact-bucket   0      2m35s
config-artifact-pvc      1      2m34s
config-defaults          1      2m33s
config-leader-election   3      2m32s
config-logging           3      2m31s
config-observability     1      2m30s
config-registry-cert     0      2m30s
feature-flags            10     2m29s
pipelines-info           1      2m28s

[bnr@bnr ckcp]$ KUBECONFIG=work/kubeconfig/admin.kubeconfig k -n default get pods
NAME                               READY   STATUS      RESTARTS   AGE
custom-env-px8m4-pod               0/1     Completed   0          102s
test-pipelinerun-bsfl5-task1-pod   0/3     Completed   0          99s
```

### CODE REVIEW

File: ckcp/run.sh [hunk 1]
```diff
@@ -97,34 +85,21 @@ else
...
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-pipelinerun.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-taskrun.yaml
-
-      # will go away with v1 graduation
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-run.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-resource.yaml
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply -f pipeline/config/300-condition.yaml
-
-      KUBECONFIG=kubeconfig/admin.kubeconfig kubectl apply $(ls pipeline/config/config-* | awk ' { print " -f " $1 } ')
+      KUBECONFIG=$KUBECONFIG_KCP kubectl apply -k ../gitops/tekton-pipeline/overlays/patched
```
Comment (Michkov) [inline ckcp/run.sh:88]: Use SCRIPT_DIR intead of relative path (everywhere).

Comment (bnallapeta) [reply ckcp/run.sh:88]: Already did that. It was kubeconfig/admin.kubeconfig earlier everywhere.. I changed that to $KUBECONFIG_KCP

Comment (Michkov) [reply ckcp/run.sh:88]: I mean instead of ../gitops/.... use $SCRIPT_DIR/gitops

**Review by Michkov — APPROVED** [review]

File: ckcp/run.sh (hunk 1 above)
Comment (bnallapeta) [reply ckcp/run.sh]: Done. Fixed all the paths.

**Review by Michkov — APPROVED** [review]

**Pull request merged**